pipeline/src/client.py

```diff
@@ -80,6 +81,7 @@ def __init__(self, read_env_data=True):
         engine_url = self._hive_engine.url
         self.flags = Flags(engine=engine_url)
         self.service_periods = Service_Periods(engine=engine_url)
+        self.flag_lookup = None
 
     #######################################################
```

Comment:
Since this is public, please add it to the client documentation in the README.md

Reply:
Will do this evening.

Reply:
And the method init_flag_dict() to please. Can you also make it very clear there that flag_lookup is not guaranteed to be non-null? A field, especially if it's public, that isn't initialized on construction just smells very off to me, and the least we can do it make sure the user knows about it.